<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>370</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Sensor Reading</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/control/play.svg</normaloff>:/control/play.svg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="1" column="0" colspan="4">
    <layout class="QHBoxLayout" name="configLayout"/>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="switcher">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Data Logger</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Initialize</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="log_2">
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QStackedWidget" name="monitors">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gaugeLayout">
         <property name="spacing">
          <number>3</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>2</number>
       </property>
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>SINE FIT</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="pauseBox">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>DAMPED SINE FIT</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="PlotWidget" name="graph"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>log_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>changeRange(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>initialize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>switcher</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>next()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>sineFit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>pause(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>dampedSineFit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeRange(bool)</slot>
  <slot>initialize()</slot>
  <slot>next()</slot>
  <slot>sineFit()</slot>
  <slot>pause(bool)</slot>
  <slot>dampedSineFit()</slot>
 </slots>
</ui>
